src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -148,13 +148,36 @@ class SignUpModalController {
             }
             onSuccess(postData)
           }
+        },
+        // TODO: make sure that this callback is only called after the sign up completes successfully
+        postSubmit: (postData, onSuccess) => {
```

Comment:
It looks like this is fired after the Registration API is fired to Okta, but before registration is completed. I think that is okay, bt as you said, we will need to check